AM_HW9_Notebook.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>How do you know the distance to your department? In my "opinion", we live on a sphere, so"direct" can have multiple meanings. Eg a straight line (probably needing a tunnel, or at least a ditch) or a circle segment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>